### PR TITLE
Fix release and version change for marquez_common python module

### PR DESCRIPTION
**Pull request by collado-mike**
Signed-off-by: Michael Collado <mike@datakin.com>

### CODE REVIEW

**Comment by mobuchowski** [comment]
Tried that approach and hit a wall here with pip20.3> dependency resolution - in that PR moved to having fake package and explicitely copying files to target packages:

https://github.com/MarquezProject/marquez/pull/1301/files

File: integrations/airflow/Dockerfile.tests [hunk 1]
```diff
@@ -2,10 +2,12 @@ FROM python:3.6-slim
 RUN mkdir /app
 ADD airflow/setup.py /app
 ADD airflow/README.md /app
-COPY common/marquez /app/marquez
+COPY common /tmp/marquez-integration-common
+RUN cd /tmp/marquez-integration-common && ls . && pip wheel --wheel-dir=/tmp/marquez-integration-common/wheel .
```
Comment (wslulciuc) [inline integrations/airflow/Dockerfile.tests:6]: _Minor_: Do you still need run the `ls .` cmd?

File: integrations/airflow/setup.py [hunk 2]
```diff
@@ -20,9 +20,10 @@
     readme = readme_file.read()
 
 requirements = [
-    "attrs>=19.3",
+    "attrs==19.3.0",
```
Comment (wslulciuc) [inline integrations/airflow/setup.py:23]: I'm wondering if we want to fix this version?

**Review by wslulciuc — APPROVED** [review]
LGTM 👍

File: integrations/airflow/Dockerfile.tests (hunk 1 above)
Comment (collado-mike) [reply integrations/airflow/Dockerfile.tests:6]: nope

File: integrations/airflow/setup.py (hunk 2 above)
Comment (collado-mike) [reply integrations/airflow/setup.py:23]: Sorry- that was done to verify the cause of a particular problem. I think we can get away without this

**Pull request merged**